nbbuild/antsrc/org/netbeans/nbbuild/extlibs/DownloadBinaries.java

```diff
@@ -161,6 +169,13 @@ public void execute() throws BuildException {
                             } else {
                                 success &= fillInFile(hashAndFile[0], hashAndFile[1], manifest, () -> legacyDownload(hashAndFile[0] + "-" + hashAndFile[1]));
                             }
+                            if (report != null) {
+                                Files.write(
+                                        report.toPath(),
+                                        (hashAndFile[0] + ";" + hashAndFile[1] + ";" + include + "\n").getBytes(),
```

Comment:
Please use the `getBytes(Charset)` variant. Depending on the default file encoding can lead to bugs.